index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Vite + React</title>
```

Comment:
Change to something like "Rating Component" or "Rating System". 

This is the title of tab in the browser, like this one 👇

![image](https://github.com/karinebznuni/rating-system/assets/91960848/4e3e0bf8-960c-4eab-8f0a-148cae0228a0)